lms/envs/common.py

```diff
@@ -2050,3 +2050,25 @@
 SEARCH_ENGINE = None
 # Use the LMS specific result processor
 SEARCH_RESULT_PROCESSOR = "lms.lib.courseware_search.lms_result_processor.LmsSearchResultProcessor"
+
+######################## Leaner profile configuration ###########################
+
+# Default visibility level for accounts without a specified value
```

Comment:
I was thinking this setting would be more for a single profile configuration (because we know we may want to support multiple in the future), what is the default visibility, what are the fields that are in the profile, and which fields are always visible. I'd do it as a dict instead of 3 different settings.